src/cpp/ext/gsm/metadata_exchange.cc

```diff
@@ -56,6 +69,8 @@ constexpr absl::string_view kMetadataExchangeLocationKey = "location";
 constexpr absl::string_view kMetadataExchangeProjectIdKey = "project_id";
 constexpr absl::string_view kMetadataExchangeCanonicalServiceKey =
     "canonical_service";
+// The keys that will be used for the local attributes when recording metrics.
+constexpr absl::string_view kMeshIdAttribute = "gsm.mesh_id";
```

Comment:
In a separate PR, we need to change all occurrances of the string "gsm" to "csm".  This includes attribute names, file and directory names, BUILD target names, etc.

Reply:
Yes!